developer-docs-site/docs/guides/use-aptos-explorer.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Using Aptos Explorer"
+slug: "use-aptos-explorer"
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Using Aptos Explorer
+
+The [Aptos Explorer](https://explorer.aptoslabs.com/) lets you delve into the activity on the Aptos blockchain in great detail, seeing transactions, validators, and account information. With the Aptos Explorer, you may ensure the work you do in the [Aptos Petra Wallet](install-petra-wallet.md) and elsewhere on the blockchain is accurately reflected in Aptos.
```

Comment:
It might be better to write 'Aptos Petra Wallet' or other wallets (without calling out which they are)

Reply:
I rewrote as, "With the Aptos Explorer, you may ensure the work you do in the [Aptos Petra Wallet](install-petra-wallet.md) and other wallets and applications is accurately reflected in Aptos."

Let me know what you think